### ISSUE TITLE
add no-overwrite

### ISSUE THREAD

**Issue by GD-Aichi** [dotansimha/graphql-code-generator#185]
I need the Option no-overwrite in the cli.
```diff
export const initCLI = (args): any => {
  commander
    .version(require('../../../package.json').version)
    .usage('gql-gen [options]')
+   .option('-ow, --no-overwrite', 'Output file(s) do not overwrite')
```
```diff
+ const fileExists = (filePath) =>{
+   try {
+    return fs.statSync(filePath).isFile();
+   } catch (err) {
+    return false;
+  }
generationResult.forEach((result: FileOutput) => {
+ if (!options.overwrite && fileExists(result.filename)) {
+   return;
+ }
  fs.writeFileSync(result.filename, result.content);
  console.log(`Generated file written to ${result.filename}`);
});
```

**Comment by dotansimha** [comment]
Sounds reasonable. @GD-Aichi , maybe you can create a PR for it?

**Comment by dotansimha** [comment]
@GD-Aichi nevermind, i'm adding it now :D

**Comment by dotansimha** [comment]
Fixed in 0.8.17 :)
